src/schema-generation/order-by-and-pagination-augmentation.ts

```diff
@@ -256,7 +298,7 @@ export class OrderByAndPaginationAugmentation {
                     });
                 }
 
-                return new TransformListQueryNode({
+                const transformListNode = new TransformListQueryNode({
```

Comment:
Do I need to add the ResultValidator in the `getPaginatedNodeUsingMultiIndexOptimization`  function too?

In all other cases maxCount needs to be undefined so I guess I can omit the ResultValidator there

Reply:
> Do I need to add the ResultValidator in the `getPaginatedNodeUsingMultiIndexOptimization` function too?

we need to do it when this function is called, but I think you can still do it outside of that function (so you don't duplicate the logic). I think the function is just a drop-in replacement for the new TransformListQueryNode(...)